main/test/ca/uwaterloo/flix/library/LibrarySuite.scala

```diff
@@ -60,4 +63,18 @@ class LibrarySuite extends Suites(
   new FlixTest("Core/Io/TestFile", "main/test/ca/uwaterloo/flix/library/Core/Io/TestFile.flix")(Options.TestWithLibAll),
   new FlixTest("Core/Io/TestInputStream", "main/test/ca/uwaterloo/flix/library/Core/Io/TestInputStream.flix")(Options.TestWithLibAll),
   new FlixTest("Core/Io/TestOutputStream", "main/test/ca/uwaterloo/flix/library/Core/Io/TestOutputStream.flix")(Options.TestWithLibAll),
+  new FlixTest("TestEnvironment", "main/test/ca/uwaterloo/flix/library/TestEnvironment.flix")(Options.TestWithLibAll) with BeforeAndAfterAll {
+    val pathEnv = java.nio.file.Files.createTempFile("test-env-",".properties");
+
+    override def beforeAll() = Try {
+      val prop = new java.util.Properties()
+      System.getenv().asScala.map{
+        case (key, value) => prop.setProperty(key, value)
+      }
+      prop.store(new java.io.FileOutputStream(pathEnv.toFile), "Test env property file")
+      System.setProperty("TEST_ENV_FILE", pathEnv.toString)
+      System.setProperty("TEST_PROPERTY", "Hello world!")
+    }
+    override def afterAll() = Try(pathEnv.toFile.delete())
+  },
```

Comment:
I think this is way overboard. I am generally in favor of testing, but in this case we are literally just "call forwarding", so I would remove the above. 

I appreciate the effort though- but in this case I don't think we should do it.

Reply:
ok, I was also exploring